src/coreclr/vm/peimage.cpp

```diff
@@ -141,11 +141,14 @@ ULONG PEImage::Release()
         result=InterlockedDecrement(&m_refCount);
         if (result == 0 )
         {
-            LOG((LF_LOADER, LL_INFO100, "PEImage: Closing Image %s\n", m_path.GetUTF8()));
+#ifdef LOGGING
+            SString path{ m_path };
```

Comment:
This is NOTHROW method. Is this going to hit contract violation? (The problem was there even before this change, but it is going to happen in more situations now.)